core/src/main/java/org/apache/calcite/sql/SqlUnnestOperator.java

```diff
@@ -121,14 +121,7 @@ private static boolean allowAliasUnnestItems(SqlOperatorBinding operatorBinding)
 
   @Override public void unparse(SqlWriter writer, SqlCall call, int leftPrec,
       int rightPrec) {
-    if (call.operandCount() == 1
```

Comment:
I tried other databases that support Unnest, and they all require two parentheses.

Reply:
Thanks for researching and fixing this @NobiGo, +1 if Calcite is the only exception, unless there is a good reason to, we generally try to comply with all major DBs.

May I only suggest to put this information in the Jira ticket description (the list of databases you tried), it might come handy later and there is less chance people will dig this information up from the PR.

Reply:
Thanks for the review @asolimando . I have added some conclusions from the research to Jira. I hope it is helpful.